caikit/config/config.yml

```diff
@@ -191,10 +198,18 @@ runtime:
 
     # Training configuration for server
     training:
-        # The directory to save trained models in
+        # These two configs are used when no `output_target` is sent with the training request
+        # They are used to construct a `Local` model saver + output target
         output_dir: training_output
         save_with_id: true
 
+        # TODO: delete?
```

Comment:
Ah- yeah it's unused now.

But I do wonder if we need some more configurability in `runtime` somewhere, otherwise the name and numbering of the output target fields are just gonna match what's in `model_management.savers`